src/output/graylogFormat.js

```diff
@@ -17,7 +17,7 @@ const formatLogs = (logs) => {
   });
 };
 export default (transaction, decodedTransaction, decodedLogs) => ({
-  networkId: networksById[getEtherNetworkId()],
+  networkId: transaction.net,
```

Comment:
This was causing all the problems related with connection loss even after getting all needed info